targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_MIMXRT1050/TARGET_EVK/fsl_clock_config.c

```diff
@@ -1,8 +1,11 @@
 /*
+ * The Clear BSD License
```

Comment:
Making a note here that this is compatible with our Apache 2 licensing.

https://spdx.org/licenses/BSD-3-Clause-Clear.html